packages/store/src/services/store/utils/file-uploader.ts

```diff
@@ -64,21 +66,26 @@ export class FileUploader {
     return resourceUrl
   }
 
-  private validateSqliteFile(filePath: string): void {
+  private async validateSqliteFile(filePath: string): Promise<void> {
     try {
-      const stats = statSync(filePath)
-
-      if (!stats.isFile()) {
+      if (!fileExistsSync(filePath)) {
+        throw new ValidationError(ErrorCodes.FILE_NOT_FOUND, {filePath})
+      }
+      if (await isDirectory(filePath)) {
         throw new ValidationError(ErrorCodes.NOT_A_FILE, {filePath})
       }
+      const sizeOfFile = await fileSize(filePath)
+      outputDebug(`Validating SQLite file at ${filePath} with size ${sizeOfFile} bytes`)
 
-      if (stats.size === 0) {
+      if (sizeOfFile === 0) {
         throw new ValidationError(ErrorCodes.EMPTY_FILE, {filePath})
       }
 
-      if (stats.size > 5 * 1024 * 1024 * 1024) {
+      if (sizeOfFile > this.MAX_FILE_SIZE) {
         throw new ValidationError(ErrorCodes.FILE_TOO_LARGE, {
-          sizeGB: Math.round(stats.size / 1024 / 1024 / 1024),
+          filePath,
+          fileSize: `${Math.round(sizeOfFile / 1024 / 1024)}MB`,
+          maxSize: `${Math.round(this.MAX_FILE_SIZE / 1024 / 1024)}MB`,
```

Comment:
```suggestion
          fileSize: `${Math.round(sizeOfFile / 1024 / 1024)} MB`,
          maxSize: `${Math.round(this.MAX_FILE_SIZE / 1024 / 1024)} MB`,
```
I think we should have a space here; there are other places in this PR (I commented on one) where we do not put a space between this

I am also curious if the CLI already has a good way to turn byte values into nice human readable ones but I guess not? Always formatting in MB might not be right depending on the scale but it will work for now.